docs/data/base/getting-started/overview/overview.md

```diff
@@ -1,22 +1,23 @@
 # MUI Base - Overview
 
-<p class="description">MUI Base is a library of blank ("unstyled") React UI components and low-level hooks.</p>
+<p class="description">MUI Base is a library of headless ("unstyled") React UI components and low-level hooks.</p>
 
 ## Introduction
 
 MUI Base is a library of unstyled React UI components.
-It features the same robust engineering as [Material UI](/material-ui/getting-started/overview/), but without implementing Material Design.
+The components were extracted from [Material UI](/material-ui/getting-started/overview/).
```

Comment:
To give more context on the origin. I think that it's a key selling point. First it means that the logic has been battle-tested and second that people will find common behaviors.